gravatar-ui/src/test/java/com/gravatar/gravatar/ui/ProfileTestUtils.kt

```diff
@@ -18,24 +18,28 @@ internal fun completeProfile(
             serviceLabel = "Mastodon"
             url = URI("https://example.com")
             serviceIcon = URI("https://example.com/icon.svg")
+            isHidden = false
```

Comment:
There are a few more places where this line has to be added. Previews are broken. 

BTW. This is why I have some doubts about this builder pattern. Offtopic though.

Reply:
Yeah. Good catch. I think I've fixed all of them.